django/db/backends/postgresql/features.py

```diff
@@ -84,6 +84,19 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         },
     }
 
+    @cached_property
+    def django_test_expected_failures(self):
+        excepted_failures = set()
+        if self.connection.use_server_side_binding:
+            # Parameters passed to expressions in SELECT and GROUP clauses are
+            # not recognized as the same values when using server-side binding
+            # cursors, see #34255.
+            excepted_failures.add(
```

Comment:
`expected_failures.update({` is more future proof in case of future additions, but whatever you think.

Reply:
add -> update